scripts/copy_bundles.py

```diff
@@ -32,6 +32,9 @@
     config,
     require,
 )
+from azul.deployment import (
+    AWS,
```

Comment:
```suggestion
    aws,
```
Repeat in all places `AWS` is imported.